Codeinwp/neve
# Fix/submenu long items wrap

### Summary

I changed the style for the submenu items on mobile to account for very long menu items.
Item names now should wrap instead of overflowing.


### Will affect the visual aspect of the product

YES

### Screenshots 
![image](https://github.com/Codeinwp/neve/assets/23024731/362fdc56-3b80-445f-b055-6fd96663f471)

### Test instructions

1. On a fresh instance with Neve
2. Create a menu where item names are very long
3. Check that all long item names are wrapped and displayed on multiple lines if required. 

## Check before Pull Request is ready:



Closes #4147.
